app/controllers/alpacas_controller.rb

```diff
@@ -1,13 +1,42 @@
 class AlpacasController < ApplicationController
 
   def index
-    @alpacas = Alpaca.all
+    @alpacas = Alpaca.where(for_sale: false) || Alpaca.where(for_sale: nil)
```

Comment:
Why for_sale: nil? I think we should add constraint so that for_sale can't be nil